app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="show-container">
+  <div class="show-header"><%= @mechanic.name %></div>
+  <div class="show-information">Years of experience: <%= @mechanic.years_experience %></div>
+  <div class="show-information">
+    <div class="show-information-header">Rides this mechanic is working on:</div>
+    <ol id="mechanic-rides">
+      <% @mechanic.rides.each do |ride| %>
+        <li>
+          <%= ride.name %>
+        </li>
+      <% end %>
+    </ol>
+  </div>
+  <%= form_with url: "/ride_mechanics", method: :post, class: "add-ride-form" do |form| %>
+    <div class="form-header">Add a ride to workload:</div>
+    <%= form.label :ride_id, "Ride ID: " %>
+    <%= form.text_field :ride_id %>
+    <%= form.hidden_field :mechanic_id, value: @mechanic.id %>
+    <%= form.submit "submit" %>
+  <% end %>
```

Comment:
I'm going to call this form not implemented properly according to the route format that we expect at this stage of Mod 2. This path should be `"/mechanics/:id/rides"`, this eliminates the need for having to pass in a hidden field and makes your route more RESTful by showing the relationship between the two entities. While we would have called your current route ok earlier on, since the introduction of nested routes, we emphasize this new pattern moving forward. 

If you're having difficulty understanding nested routing, please reach out. I will not be taking points this time.